lib/msf/core/payload/adapter/fetch/tnftp.rb

```diff
@@ -0,0 +1,12 @@
+module Msf::Payload::Adapter::Fetch::Tnftp
```

Comment:
It might be better to do the `which` trick in the ftp payload, but the problem is that ftp might exist as the interactive-only ftp.